Dockerfile

```diff
@@ -1,25 +1,49 @@
-FROM rust:1.44.1-slim-buster
+FROM debian:buster-slim
+RUN apt-get update
 
-RUN apt-get update \
-  && apt-get install -y --no-install-recommends clang=1:7.* cmake=3.* \
-     libsnappy-dev=1.* \
-  && apt-get clean \
-  && rm -rf /var/lib/apt/lists/*
+### Bitcoin ###
 
-RUN adduser --disabled-login --system --shell /bin/false --uid 1000 user
+# Buildtime dependencies
+RUN apt-get install -qqy build-essential libtool autotools-dev automake pkg-config bsdmainutils python3
 
-WORKDIR /home/user
-COPY ./ /home/user
-RUN chown -R user .
+# Runtime dependencies
+RUN apt-get install -qqy libevent-dev libboost-system-dev libboost-filesystem-dev libboost-test-dev libboost-thread-dev
 
-USER user
+# Clone source
+RUN apt-get install -qqy git
+WORKDIR /build/bitcoin
+RUN git clone --branch locations https://github.com/romanz/bitcoin.git .
 
-RUN cargo install --path .
+# Build
+RUN ./autogen.sh && ./configure --disable-wallet --without-gui --without-miniupnpc && make -j"$(($(nproc)+1))"
```

Comment:
Uff, building bitcoin for every test? That looks quite heavy. What about using my repository?

Reply:
IIUC, it should be cached between subsequent builds of `electrs`.

Reply:
Ah, OK. If there's any problem with that, just ping me and I'll look into integrating my repo.